models/intel/bert-large-uncased-whole-word-masking-squad-emb-0001/description/bert-large-uncased-whole-word-masking-squad-emb-0001.md

```diff
@@ -60,14 +60,14 @@ The values are equal to:
 * `[SEP]` is a special separator token inserted between the question and premise of the question.
 * `[PAD]` is a special token used to fill the rest of the input to get given input length (384).
 
-4. Position indexes,  name: `position_ids`, shape: [1x384] for context and [1x32] for question.
+4. Position indexes,  name: `position_ids`, shape: `1, 384` for context and `1, 32` for question.
 Position indexes are sequence of integer values from 0 to 383 (or 31 for question) representing the position index for each input token. The `position_ids` is always the same for any input tokens set
 
 ## Output
 
 The outputs of the net is `1, 1024` floating point-valued embedding vector. These vectors can be used to find better context with answer to the question by simple comparing the context embedding vector with question context embedding vector in 1024D embedding space.
 
-1. Embedding: name: `embedding`, shape: [1x1024].
+1. Embedding: name: `embedding`, shape: `1, 1024`.
```

Comment:
as I know we use bullet points for single value not numeric list
```suggestion
* Embedding: name: `embedding`, shape: `1, 1024`.
```

Reply:
Updated